src/components/NamespaceFilter/NamespaceFilter.tsx

```diff
@@ -68,8 +68,9 @@ export class NamespaceFilter extends React.Component<NamespaceFilterProps, Names
         this.setState({ filterTypeList: initialFilters });
       })
       .catch(error => {
-        console.error(error);
-        this.props.onError('Error fetching namespace list.');
+        const errMsg = API.getErrorMsg('Error fetching namespace list.', error);
```

Comment:
I imagine this one doesn't come from axios, isn't it ?

Reply:
Yes it does, as it's originated from `API.getNamespaces` so an axios call